### ISSUE TITLE
Copy and pasting clipboard not working on my world

### ISSUE THREAD

**Issue by wa04sy** [TEdit/Terraria-Map-Editor#1811]
**What is the problem?**
- I am unable to paste any schematics previously saved schematics or freshly copied ones into my specific terraria world

**What I expected to happen?**
- The previously saved schematic or a freshly made one to be pasted into the map where I pointed it, but it did not work for my specific world

**Screenshots:**
- What should've happened in a regular world (Happening in a world generated by TEdit, same in any other terraria world) (Pasting anywhere I wanted):
![image](https://user-images.githubusercontent.com/69479494/202707620-5641d78e-5396-48af-85ce-da723b2a6ec2.png)

**Desktop:**
 - OS: Windows 11
 - Version TEdit v4.11.8
 - Terraria Version 1.4.4.9

TL;DR I'm unable to copy and paste clipboard content in TEdit onto a specific terraria world of mine, while any other world works normally.

**Comment by RussDev7** [comment]
This should be resolved by updating to the latest TEdit version.
